unit-deployment/dnscrypt-server/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  dnscrypt-server:
+    container_name: dnscrypt-server
+    ports:
+      - 5443:5443/udp
+      - 5443:5443/tcp
+    volumes:
+      - /LOCAL_CUSTOM_DNSCRYPT_SERVER_CONF_DIR/unbound-conf:/opt/unbound/etc/unbound/zones
+      - /LOCAL_CUSTOM_DNSCRYPT_SERVER_CONF_DIR/keys:/opt/encrypted-dns/etc/keys
+    image: nyvanga/dnscrypt-server
```

Comment:
I would like to use a tag here. So that is as of now: `:2022-09-22` I see on your Docker Hub. Plz 🙏 

---

The reason. It's a security "product" so it's nice to know what you're eating "for breakfast". Thanks

Reply:
Yeah well, as previously mentioned, I do not plan to maintain the image (since it builds for more than 1 hour each time, and I don't want to spend my free Github actions minutes on that). So don't know how to fix this.

Reply:
Add `:2022-09-22` to the image tag and we gooood. Then the next time it runs ... when you see fit and your free GitHub credits available allows it ... then cool.

Reply:
Wonderful! That's really awesome!